saos-batch/src/test/java/pl/edu/icm/saos/batch/core/indexer/JudgmentReindexingJobTest.java

```diff
@@ -101,7 +98,7 @@ public void judgmentReindexingJob_ONLY_SC_JUDGMENTS() throws Exception {
         JobParameters jobParameters = new JobParameters(ImmutableMap.of("sourceCode", new JobParameter(SourceCode.SUPREME_COURT.name())));
         JobExecution jobExecution = jobLauncher.run(judgmentReindexingJob, jobParameters);
         
-        assertStepWriteCount(jobExecution, 0, SUPREME_COURT_JUDGMENTS_COUNT);
+        JobExecutionAssertUtils.assertJobExecution(jobExecution, 0, SUPREME_COURT_JUDGMENTS_COUNT);
         solrJudgmentsServer.commit();
         
         assertAllMarkedAsIndexed();
```

Comment:
It would be nice if you checked if all the indexed judgments are SupremeCourtJudgments. Checking only the number doesn't tell you it.